nixos/extractors/services.nix

```diff
@@ -140,6 +166,14 @@ in {
           }));
         };
 
+      nextcloud = mkIf config.services.nextcloud.enable {
+        name = "Nextcloud";
+        icon = "services.nextcloud";
+        details.listen.text = "http${
```

Comment:
Since this will actually be the user-facing URL, I'd propose to set the same value on `info` instead of `details.listen`.

Reply:
Agree, implemented via https://github.com/oddlama/nix-topology/pull/4/commits/e4d4a4bd62901801b956f5e1dfa241ca94cd3711